context/app/static/js/components/home-revision/FacetSearchEntityListItems/FacetSearchEntityListItems.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { StyledList, FacetLabel, FacetValue } from './style';
+
+function FacetSearchEntityListItems({ entityType, matches, labels }) {
+  return (
+    <li>
+      {Object.entries(matches).map(([k, v]) => {
+        return (
+          <>
+            <FacetLabel color="primary" variant="subtitle2">{`${labels[k]} (${entityType}s)`}</FacetLabel>
+            <StyledList>
+              {v.map((m) => {
+                const facetQueryParam =
+                  k === 'mapped_data_access_level'
+                    ? `mapped_status-mapped_data_access_level[0][0]=Published&mapped_status-mapped_data_access_level[1][0]=${encodeURIComponent(
```

Comment:
What's the motivation for this section? Is it to make sure we only show published entities when you're logged in? The dashed param names are there for the hierarchical facets on search... but I'd like to avoid baking in dependencies on those details.

Reply:
I added this to ensure what we link to mirrors what is accessible from the facet UI on the search page.

Reply:
This is so the corresponding checkboxes/inputs are selected and you can reverse the selection from the UI. Otherwise we be sending them to a page which can only be altered by the url query params.

Reply:
Oh, I get it: I wasn't thinking off the publication status as something the homepage search would cover. Thanks for being thorough... but I'm going to file a follow-up to suggest we pull this out: It's not unlikely that that facet will change, and things that need to be modified in tandem across a codebase make me nervous.